package.json

```diff
@@ -4,7 +4,8 @@
   "description": "This is a Workshop about Cypress",
   "main": "index.js",
   "scripts": {
-    "test:open": "cypress open"
+    "test:open": "npx cypress open",
+    "test:run": "npx cypress run"
```

Comment:
the npx command is not needad as prefix for test:open or test:run scripts

Reply:
Checking progress with Juan De la Hoz, "cypress open" command didn't work though we used npx and it did.   I will check if woks as you suggested.   If not I will ask on Monday.

Reply:
Fixed, I will send changes for you to check

Reply:
Hi Guys, so, like we discuss, I suggest Jorge continue in JS, cause the TS maybe is missing dependencies.

Reply:
Additionally, the command Cypress sometimes is not recognized like a global command, so is possible to use the 'npx' too in the command